requirements.txt

```diff
@@ -1,7 +1,7 @@
 boto3==1.9.246
 botocore==1.12.246
 coloredlogs==14.0
-docutils==0.15
+docutils<0.16,>=0.14
```

Comment:
what version do we actually hit?

Reply:
It gets the pre-release versions, something like 0.17.1b1.dev0.